### ISSUE TITLE
Need to provide a way for agencies to be able to upload just trailheads

### ISSUE THREAD

**Issue by ryanbranciforte** [codeforamerica/OpenTrails#108]
Currently, an agency must upload a trail file in order to get to the trailheads.

**Comment by migurski** [comment]
Hm. The actual trails are a required part of the spec, so an OT process without them would be an error. I suppose an explicitly empty segments file would be okay, though weird.

**Comment by ryanbranciforte** [comment]
agree, it's odd re requirements of OT, but the converter does offer the ability to skip trailheads, but not skip trails.
